### ISSUE TITLE
temporarily_unavailable Status:Fail While Registering

### ISSUE THREAD

**Issue by Johnprogg** [tgalal/yowsup#2961]
Hey any Genius Expert Can Fix this problem, Getting this error After Registering...!!
```
W 2020-04-01 16:00:05,992 yowsup.common.http.warequest - Passing Config to WARequest is deprecated, pass a YowProfile instead
I 2020-04-01 16:00:06,901 yowsup.common.http.warequest - {"login":"18043754166","status":"fail","reason":"temporarily_unavailable","retry_after":3600}

status: fail
retry_after: 3600
login: 180437XX166
```

**Comment by modcastro** [comment]
This usually occurs when the IP is blocked due to excessive requests.

**Comment by Johnprogg** [comment]
Hey @modcastro Thank You So Much, Have you recently Registered any WhatsApp Number ? Does it worked ?

**Comment by fevzican** [comment]
Same to, I tried with new IP and it was banned.

**Comment by Johnprogg** [comment]
Where you are Deploying This @fevzican ?

**Comment by Johnprogg** [comment]
@modcastro Please Help us our Numbers are getting banned and showing 'status":"fail","reason":"temporarily_unavailable" i already tried with 4 numbers with all the different IP's i am using Ubuntu EC2 Instance's and also @fevzican tried 10 different numbers and 5 servers we are constantly getting failed, Please help us what changes we can make to register sucessfully

**Comment by jamescder** [comment]
Hey, I am also Facing the same issue as you guys @modcastro recently seen your reply you have successfully deployed 3 Yowsup Registrations can you please explain what steps you have followed ?
It would be great if you just post the exact procedure you have done, Like where you deployed and what number you are using to register etc.
please help us all !!

**Comment by fevzican** [comment]
> ISP

With DigitalOcean, What do you use ?
Actually I could register yesterday some numbers, firstly tried with my phone.

**Comment by fevzican** [comment]
> Where you are Deploying This @fevzican ?
> Have you changed your ISP and tried?

With DigitalOcean, What do you use ?
Actually I could register yesterday some numbers, firstly tried with my phone.

**Comment by Johnprogg** [comment]
It's working for you now ? @fevzican

**Comment by fevzican** [comment]
> It's working for you now ? @fevzican

Nope, tried an hour ago. Still geting banned with yowsup, only works after registration with my phone

**Comment by Johnprogg** [comment]
@fevzican I think Yowsup Started to Deduct Virtual Servers may be

**Comment by fevzican** [comment]
> @fevzican I think WhatsApp Started to Deduct Virtual Servers may be, Is there any modification we can do to fix this ?
> 
> Between which mcn and cc are you using ?
> 
> Are you trying to Register it with Completely new number ?

This problem is not about with phone numbers I think, I tested on different countries. I think it caused by servers or yowsup. Maybe, I can try with some unknown local servers, then we can sure it is about yowsup. In the end, we send always same info like device name, model etc, is not it?

**Comment by Johnprogg** [comment]
@fevzican I think the same it's the issue with Device name samsung, Same Model and they are same always, I think that's the problem.

**Comment by GitUpLad** [comment]
Not working.

**Comment by Pyatka-Ubiyca** [comment]
i can reg one of 5  numbers.. :(

**Comment by LuKks** [comment]
I have it working very well, the other day I just registered a new phone number and I'm able to send text messages, etc. Although I can't send video or image yet.
Contact to me via mail and maybe I can help you.

**Comment by Pyatka-Ubiyca** [comment]
> I have it working very well, the other day I just registered a new phone number and I'm able to send text messages, etc. Although I can't send video or image yet.
> Contact to me via mail and maybe I can help you.

could you  show  your steps for success registration?
(maybe any proxies or special user agents)
